src/components/PlanRoom/SaveRoomTables/SaveBoard.tsx

```diff
@@ -0,0 +1,7 @@
+const SaveBoard = (tables: {
+  [key: string]: { top: number; left: number; text: string; width: number; height: number; borderRadius: number };
+}) => {
+  console.log(tables);
```

Comment:
I don't see any point in showing / implementing this component.
It does nothing except logging information to console, which we are not interested in.

Reply:
will be deleted